.circleci/config.yml

```diff
@@ -1187,10 +1166,6 @@ workflows:
 
       - test_deploy_pypi:
           filters:
-            branches:
-              only:
-                - /rel\/.*/
-                - /maint\/.*/
```

Comment:
@mgxd Vaguely recall you were the one who preferred to keep this off of every PR. Should I restore these filters now that we've tested?

Reply:
I think so, that'll give us more flexibility during development cycles.